addons/point_of_sale/static/src/app/store/pos_store.js

```diff
@@ -36,8 +36,20 @@ import { FormViewDialog } from "@web/views/view_dialogs/form_view_dialog";
 import { CashMovePopup } from "@point_of_sale/app/navbar/cash_move_popup/cash_move_popup";
 import { ClosePosPopup } from "../navbar/closing_popup/closing_popup";
 import { user } from "@web/core/user";
-
+import { FloorScreen } from "@pos_restaurant/app/floor_screen/floor_screen";
```

Comment:
You cannot import it from modules unrelated to depend modules.
not work when `pos_restaurant` is not installed!